### PR TITLE
Feature/get unit test to run on visual studio code

### PR DESCRIPTION
**Pull request by JeppeBlixen**
Small changes to run act platform tests in vs code project

### CODE REVIEW

File: test_requirements.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 pyspark
 packaging
 pytest
+PyYAML
+sqlparse
```
[the remark this answers is not in the document]
Comment (mrmasterplan) [reply test_requirements.txt:5]: I just checked again, they are already there. Why do you need them here?￼

File: test_requirements.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
 pyspark
 packaging
 pytest
+PyYAML
```
Comment (mrmasterplan) [inline test_requirements.txt:4]: I don't think this is necessary. please install by running

    python setup.py develop

to install all requirements.

File: test_requirements.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (JeppeBlixen) [reply test_requirements.txt:5]: I cannot run unittests with out these packages

File: test_requirements.txt (hunk 2 above)
Comment (JeppeBlixen) [reply test_requirements.txt:4]: I have run python setup.py develop, still got some issues

**Pull request merged**